src/util/dbfunctions.js

```diff
@@ -16,6 +16,14 @@ export async function displayData(calls) {
     return calls.map((call) => allCalls.find((iterator) => (call.name === iterator.name)));
 }
 
+export function namesArray(array) {
+    var namesArray = [];
+    array.forEach((element) => {
+        namesArray.push(element.name);
+    })
```

Comment:
missing ;

consider adding prettify to your code editor, it should fix things like this as you save

Reply:
@cdb1212 you're still missing a semi-colon at the end of the forEach